### ISSUE TITLE
Implement network score module from cord.js in Golang

### ISSUE THREAD

**Issue by NiranjanAP** [cord-network/community#5]
Refer https://github.com/dhiway/cord.js/tree/develop/packages/network-score, this is the network scoring module. This needs to be built in golang to help some customers who have golang as their primary development language.

Note: You'll have to identify and build some of the other modules that are dependencies for this to work, hence a high level of understanding of the SDK will help. 
To get a good overview of the the SDK, Go through the demo script https://github.com/dhiway/cord.js/blob/develop/demo/src/func-test.ts and documentation for the same at https://docs.cord.network/cordjs/integratecord/

**Comment by amarts** [comment]
depends on #2  Once the basic core modules are complete, this can be done as a separate task.
